MonoGame.Framework/TitleContainer.cs

```diff
@@ -3,7 +3,11 @@
 // file 'LICENSE.txt', which is part of this source code package.
 
 using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Net.Http;
+using System.Net.Sockets;

```

Comment:
Not all console platforms have working networking at the C# level.

We need to put any networking into platform calls.  The default implementation for PC/mobile is free to use C# networking APIs.  For console it is a different story.